docs/esp32/tutorial/peripheral_access.rst

```diff
@@ -17,7 +17,6 @@ the prescaler of the MCPWM0 peripheral.
     DPORT_PERIP_RST_EN_REG = const(DR_REG_DPORT_BASE + 0x0C4)
     DPORT_PWM0_CLK_EN = const(1 << 17)
     MCPWM0 = const(0x3FF5E000)
-    MCPWM1 = const(0x3FF6C000)
```

Comment:
I think it's worth keeping this here in case someone wants to adapt the example to use this instance.